src/screens/CommunityHome/index.tsx

```diff
@@ -155,9 +155,11 @@ export default function CommunityHome({ route }: any) {
         // setCommunityData
         (community) => {
           setCommunityData(community);
-          setPostSetting(community?.data?.postSetting)
-          if ((community.data as Record<string, any>).needApprovalOnPostCreation) {
-            setPostSetting('ADMIN_REVIEW_POST_REQUIRED')
+          setPostSetting(community?.data?.postSetting);
+          if (
+            (community.data as Record<string, any>).needApprovalOnPostCreation
```

Comment:
```suggestion
            (community.data as Amity.Community).needApprovalOnPostCreation
```
if possible, we should get rid of `any` when we can

Reply:
sure. removed 'any'.